rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/Sw360ModerationRequestService.java

```diff
@@ -242,102 +245,93 @@ public Map<PaginationData, List<ModerationRequest>> getRequestsByState(User sw36
      */
     public ModerationState acceptRequest(ModerationRequest request, String moderatorComment, @NotNull User reviewer)
             throws TException {
-
+        ComponentService.Iface sw360ComponentClient = getThriftComponentClient();
+        ProjectService.Iface sw360ProjectClient = getThriftProjectClient();
+        LicenseService.Iface sw360LicenseClient = getThriftLicenseClient();
+        SPDXDocumentService.Iface sw360SPDXClient = getThriftSPDXDocumentClient();
+        DocumentCreationInformationService.Iface documentCreationInfoClient = getThriftDocumentCreationInfo();
+        PackageInformationService.Iface packageInfoClient = getThriftPackageInfo();
```

Comment:
Any reason to move all of the client outside the individual `case`?

Reply:
@nikkuma7 If there is no specific reason, can we move the clients back to switch/case as in current code?